app/src/main/res/layout/about_shabd.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android" android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="100dp"
```

Comment:
Remove all harxcoded values. Use dimens, colours, strings value.

Reply:
dimensions are OK for now but the text should be taken from resources